Assets/MixedRealityToolkit.SDK/Experimental/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -23,19 +23,8 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI
     public class ObjectManipulator : MonoBehaviour, IMixedRealityPointerHandler, IMixedRealityFocusChangedHandler
     {
         #region Public Enums
-        [System.Flags]
-        public enum HandMovementType
-        {
-            OneHanded = 1 << 0,
-            TwoHanded = 1 << 1,
-        }
         public enum RotateInOneHandType
         {
-            MaintainRotationToUser,
-            GravityAlignedMaintainRotationToUser,
-            FaceUser,
-            FaceAwayFromUser,
-            MaintainOriginalRotation,
             RotateAboutObjectCenter,
```

Comment:
Would it be possible to leave a comment stating what sort of things should be added here in the future, vs being made separate constraints? i.e. if we're going to cull this and leave stuff in here, we should write down our philosophy for what this should be compared to constraints.